src/sentry/api/serializers/models/alert_rule.py

```diff
@@ -1,7 +1,8 @@
 from collections import defaultdict
-from typing import MutableMapping
+from typing import List, MutableMapping, Optional
```

Comment:
Can you please add `from __future__ import annotations` and then use `list`?

Reply:
To my understanding `List` is used more commonly within `src/sentry/api`